vscode/src/non-stop/FixupController.ts

```diff
@@ -996,9 +996,30 @@ export class FixupController
 
         // append response to new file
         const doc = await vscode.workspace.openTextDocument(newFileUri)
-        const pos = new vscode.Position(Math.max(doc.lineCount - 1, 0), 0)
-        const range = new vscode.Range(pos, pos)
+        const lastLine = doc.lineAt(Math.max(doc.lineCount - 1, 0))
```

Comment:
```suggestion
        const lastLine = doc.lineAt(doc.lineCount - 1)
```
doc.lineAt will returns `TextLine` which includes`isEmptyOrWhitespace`, so we probably don't need to use `Math.max` here.